scripts/st2_bootstrap.sh

```diff
@@ -181,5 +187,8 @@ else
 
     echo "Running deployment script for st2 ${VERSION}..."
     echo "OS specific script cmd: bash ${BOOTSTRAP_FILE} ${VERSION} ${RELEASE} ${REPO_TYPE} ${USERNAME} ${PASSWORD}"
-    bash ${BOOTSTRAP_FILE} ${VERSION} ${RELEASE} ${REPO_TYPE} ${DEV_BUILD} ${USERNAME} ${PASSWORD}
+    TS=$(date +%Y%m%dT%H%M%S)
+    sudo mkdir -p /var/log/st2
+    bash ${BOOTSTRAP_FILE} ${VERSION} ${RELEASE} ${REPO_TYPE} ${DEV_BUILD} ${USERNAME} ${PASSWORD} 2>&1 | adddate | sudo tee /var/log/st2/st2-install.${TS}.log
+    exit ${PIPESTATUS[0]}
```

Comment:
Awesome :+1:

Reply:
I can't decide if it's good or bad that I'm learning new Bash tricks here. Probably good.